.npmignore

```diff
@@ -1,3 +1,4 @@
+appveyor.yml
```

Comment:
we're removing this file and `.travis.yml` either, so no need to be here

Reply:
good spot - removing them now :)